### ISSUE TITLE
[Android Bootcamp] Project 0: Todo - Ready for Review

### ISSUE THREAD

**Issue by sarnarahul** [sarnarahul/PreWorkTodoAndroidApp#1]
My app is complete, please review. /cc @codepathreview @codepath

I have coded the basic app for now, but can I still update it if possible before end date?

**Comment by nesquena** [comment]
Hi Rahul,

Thanks for applying and submitting your pre-work. Based on your LinkedIn it is unclear if you meet the [selection criteria for participating in the course](http://courses.codepath.com/snippets/intro_to_android/selection_criteria). In particular:
- Within past 3 years have been employed as a software engineer, team lead, technical founder, VPE
- 2+ years of professional software development experience (average participant has 5+ years)
- Sustained professional software experience at companies involving shipping production code strongly preferred

Can you review that and let me know if you can confirm that you meet the requirements? If not, I can add you as a "remote self-paced observer" so you can still have access to the curriculum and learn at your pace. In the meantime, you should also check out the [resources we provide here](http://courses.codepath.com/snippets/intro_to_android/email_faq#heading-i-dont-have-much-or-any-professional-software-development-experience-how-can-codepath-help-me).

**Comment by nesquena** [comment]
> I have coded the basic app for now, but can I still update it if possible before end date?

Yes, either way feel free to continue working on the pre-work after initial submission.

**Comment by sarnarahul** [comment]
I had been working with CA Inc. on two live projects during my last year of college, one is in production and other is about to go to market (so I am not sure if this can included in the professional software development experience) and by end of this month it will be more than one year at my current job. So I am combining both as 2+ years.

**Comment by nesquena** [comment]
Ok thanks for letting me know. The next step is to continue working on [extensions to your todo app](http://courses.codepath.com/snippets/intro_to_android/prework#heading-5-extending-your-todo). With regards to extensions to implement, we are looking for developers with an interest in user experience and product development so be sure to focus on additional functional enhancements and/or improving the user interface of the app. Please ping us back for a review once you've added a few extensions.

We will be determining final wave of admittance into the program over the next week. For those not admitted as participants, we will offer access to applicants as a remote observer cohort that has access to the materials and is able to work on the projects at their own pace. You will hear from us again soon.
